webapp/lib/platform.dart

```diff
@@ -57,10 +57,12 @@ bool isUserSignedIn() {
 
 typedef CollectionListener(List<DocSnapshot> changes);
 
-void listenForMetrics(String collectionRoot, CollectionListener listener) {
+void listenForMetrics(String collectionRoot, String collectionType, CollectionListener listener) {
```

Comment:
I suggest we change this to take a single `collectionPath` as an argument, and leaving the callers to construct the path, by calling, e.g.:
```
listenForMetrics('${PROJECTS[i]}/${NEEDS_REPLY_METRICS_COLLECTION_KEY}/metrics')
```

you should be able to then just pass the `collectionPath` argument to 
```
_firestoreInstance
    .collection(collectionPath)
    ....
```

Reply:
Okay. I didn't know I could do that. Thanks for the tip